### ISSUE TITLE
Panic after resising

### ISSUE THREAD

**Issue by Kubuxu** [ipfs/kubernetes-ipfs#10]
```
## Running test 'Simple Add and Pin'
Not enough nodes... Scaling up (pausing for 15 seconds)
Sleeping 15 seconds...
## Starting 5 nodes for this test
### Running step 'Add file' on node 1
$ head -c 10 /dev/urandom | base64 > /tmp/file.txt && cat /tmp/file.txt && ipfs add -q /tmp/file.txt
### Saving output from line 0 to variable FILE
### Saving output from line 1 to variable HASH
### Running step 'Pin file on node 2' on node 2
### Getting variable FILE
### Getting variable HASH
$ ipfs pin add $HASH > /dev/null && ipfs cat $HASH
Assertion Passed

### Running step 'Pin file on node 3' on node 3
### Getting variable FILE
### Getting variable HASH
$ ipfs pin add $HASH > /dev/null && ipfs cat $HASH
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0x530b20, 0xc42000e160)
	/usr/lib/go/src/runtime/panic.go:500 +0x1a1
main.main()
	/home/kubuxu/go/src/github.com/ipfs/kubernetes-ipfs/main.go:208 +0x10a9
exit status 2
```

It was completely fresh run.

**Comment by FrankPetrilli** [comment]
Something of a known issue, I'm not yet spinlocking on containers being in "Running" state, but stupidly pausing for 15 seconds. I'll see if I can quickly add this.

**Comment by FrankPetrilli** [comment]
Should be resolved in 960c5452a4aaff4c1a67f047ba08f301a680baf1
